spec/classes/foreman_spec.rb

```diff
@@ -252,6 +252,95 @@
         end
       end
 
+      describe 'with all parameters and Sensitive for Secrets' do
+        let :params do
+          {
+            foreman_url: 'http://localhost',
```

Comment:
You don't need to pass in all values. Just passing in those you care about should be sufficient. Then it'll use the defaults from params for the other ones.